lib/index.js

```diff
@@ -165,9 +165,54 @@ export default Adapter => class PostgreSQLAdapter extends Adapter {
 
     for (let field in options.match) {
       const value = options.match[field]
-      const isArray = fields[field][isArrayKey]
+      const isArray = (fields[field] ?
+              fields[field][isArrayKey] :
+              fields[field.split('_')[0]][isArrayKey])
+      const containsOperator = field.includes('_')
```

Comment:
For example it won't work with the runtime only, but it works when using the polyfill or executable since it overrides global prototypes.